spharpy/samplings/samplings.py

```diff
@@ -15,11 +15,12 @@
 from ._eqsp import lebedev_sphere
 
 
-def cube_equidistant(n_points):
+def equidistant_cuboid(n_points, flatten_output=False):
```

Comment:
```suggestion
def equidistant_cuboid(n_points, flatten_output=True):
```
For consistency witch other samplings I'd keep the flattening as default.